lib/timeline/component/LineGraph.js

```diff
@@ -341,7 +341,7 @@ LineGraph.prototype._onRemove = function (ids) {
   this._onUpdate(ids);
 };
 LineGraph.prototype._onUpdateGroups = function (groupIds) {
-  this._updateAllGroupData();
+  this._updateAllGroupData(null,groupIds);
```

Comment:
Yes, we are thinking about it. But a code formatter would destroy the "blame" History. It's maybe something for the next major release..